### PR TITLE
Removed lib-monitor

### PR DESCRIPTION
**Pull request by dphillips-39**
Removed Lib-monitor from inventory and lib-monitor.conf

Related to #5118

### CODE REVIEW

File: inventory/all_projects/_orphans [hunk 1]
```diff
@@ -5,7 +5,6 @@ issuetracker-prod1.princeton.edu
 lib-adc1.princeton.edu
 lib-adc2.princeton.edu
 [monitor_backends]
```
Comment (acozine) [inline inventory/all_projects/_orphans:7]: We should probably remove the group as well:
```suggestion
```

Comment (Beck-Davis) [reply inventory/all_projects/_orphans:7]: Okay, we'll remove it and do another commit on this branch when we come back to this ticket tomorrow. Thanks @acozine, I wasn't sure if we would need this group later.

Comment (dphillips-39) [reply inventory/all_projects/_orphans]: This group was removed

**Review by acozine — APPROVED** [review]
Looks good, thanks @dphillips-39

**Pull request merged**